scripts/build-engine

```diff
@@ -7,4 +7,5 @@ source $(dirname $0)/lib/version
 cd $(dirname $0)/..
 mkdir -p bin
 echo Building submariner-engine version $VERSION
-CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION" -o bin/submariner-engine main.go
+CGO_ENABLED=0 go build -ldflags "-s -w -X main.VERSION=$VERSION" -o bin/submariner-engine main.go
```

Comment:
What do you think of an env bar we could pass down via DAPPER_ENV_VARS to make, so we are able to build with full debug and without packing?

I think we could need that for development at least.